mlos_bench/mlos_bench/services/local/local_exec.py

```diff
@@ -122,8 +122,10 @@ def _local_exec_script(self, script_line: str,
         """
         cmd = shlex.split(script_line)
         script_path = self.config_loader_service.resolve_path(cmd[0])
-        if os.path.exists(script_path):
+        if os.path.exists(script_path) and not os.path.isdir(script_path):
             script_path = os.path.abspath(script_path)
+        else:
+            script_path = cmd[0]  # rollback to the original value
 
         cmd = [script_path] + cmd[1:]
```

Comment:
This is kinda convoluted the way it is now.
We first split the line.
Then we resolve the first component of it to a full path.
But then we check if that's a directory instead of a file, in which case we undo the resolution.
Then we combine the array.
But then we check if we need to expand it again ...
It just all feels kinda kludgey to me...

Reply:
I started playing with something locally - lemme play with that quickly and see if that can be used here.

Reply:
Chatted offline.  Let's do this for now and I'll push a slightly more complete fix later on.